agent/config/types.go

```diff
@@ -110,7 +110,7 @@ type Config struct {
 	TaskIAMRoleEnabled bool
 
 	// TaskCPUMemLimit specifies if Agent can launch a task with a hierarchical cgroup
-	TaskCPUMemLimit bool
+	TaskCPUMemLimit DefaultBool
```

Comment:
Does setting this from config JSON work as well? I have a feeling that `DefaultBool` may need to implement the `Unmarshaler` interface: https://golang.org/pkg/encoding/json/#Unmarshaler

Reply:
Probably not! I'll fix this